src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -205,12 +202,14 @@ class StockMovement {
 
         // Include all requisition items except those that are substitutions or modifications because the
         // original requisition item will represent these changes
-        if (requisition.requisitionItems) {
-            SortedSet<RequisitionItem> requisitionItems = new TreeSet<RequisitionItem>(requisition.requisitionItems)
-            requisitionItems.each { requisitionItem ->
-                if (!requisitionItem.parentRequisitionItem) {
-                    StockMovementItem stockMovementItem = StockMovementItem.createFromRequisitionItem(requisitionItem)
-                    stockMovement.lineItems.add(stockMovementItem)
+        if (!CH.config.openboxes.api.pagination.enabled) {
```

Comment:
Do we need this here? If yes, then go ahead and merge the PR.

Reply:
@DominikaHatala @jmiranda Hmm, I thought that we agreed, that backend won't check this property, and it will be sent in app context api and then front will decide what it want to do with it.

Reply:
@DominikaHatala Get back to us on this ASAP. If removing this block of code is going to require a large change to the front end (i.e. retrieving stock movement items in a separate request) then we should probably move this to a separate ticket. But for now let's remove the config property check here and deal with the consequences (i.e. multiple requests retrieving stock movement items).

Reply:
We create stock movement from requisition not only for react pages but also for old pages(e.g stock movement list page). If we remove adding stock movement items here, the number of items displayed on this page will always be 0. If we leave it without any condition, line items will always be added even if there is pagination enabled. Maybe we should add a boolean isPaginated argument to getStockMovement(String id) method and set stockMovement.lineItems null if it's true? We will send isPaginated in StockMovementApiController based on max/offset parameters. What do you think @awalkowiak @jmiranda ?

Reply:
@DominikaHatala @jmiranda if this change will require some further bigger changes in old front end, then I think we can skip it and make a separate ticket for refactor once we have done api and react front.

Reply:
@DominikaHatala AFAIK the stock movement list page doesn't use the API. Are there other pages that do? But in that case, I don't mind including lineItems on every request then. The lineItems aren't the performance bottleneck. It's when we get availableItems, picklistItems, and substitutionItems on the edit, pick and pack pages that causes the issue. So get rid of the config property check and allow the `/api/stockMovement/:id` response to include lineItems. 

@awalkowiak Sounds good to me re: refactoring in a separate ticket.

Reply:
@DominikaHatala Could you create a new ticket for this and try to enumerate all (or at least a few) of the places that access the `lineItems` attribute of the `/api/stockMovement` response.

Reply:
@DominikaHatala And I see what you meant by this 

> We create stock movement from requisition not only for react pages but also for old pages(e.g stock movement list page). 

The stock movement list page is not going through the API but it calls the same transform/translate methods that the API does when going from Requisition to Stock Movement. 

So yes we should (at least for now) 
1) Always include lineItems when retrieving a stock movement object even if pagination is enabled
2) Create a ticket to consider moving away from including lineItems (although I'm starting to suspect there might not be a need for this).

Reply:
@jmiranda For now we always include lineItems and it seems that there is no need for moving it away. I think that pagination on backend is finished but I would like to wait with merging it until pagination on frontend is finished.

Reply:
@DominikaHatala Sounds good to me.